worker/operatorhttpserver/server.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2020 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package operatorhttpserver
```

Comment:
This won't always necessarily just be for an operator.
Maybe call it simplehttpserver or muxhttpserver or something